components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint.util/src/main/java/org/wso2/carbon/identity/mgt/endpoint/util/client/api/RecoveryApiV2.java

```diff
@@ -96,10 +103,58 @@ public List<AccountRecoveryType> initiatePasswordRecovery(RecoveryInitRequest re
     public RecoveryResponse recoverPassword(RecoveryRequest recoveryRequest, String tenantDomain,
                                             Map<String, String> headers) throws ApiException {
 
-        String localVarPath = "/password/recover".replaceAll("\\{format\\}", "json");
+        final String localVarPath = "/password/recover";
         return recover(recoveryRequest, tenantDomain, headers, localVarPath);
     }
 
+    /**
+     * Resend password recovery confirmation details.
+     *
+     * @param resendRequest   Resend request. (required)
+     * @param tenantDomain    Tenant Domain which user belongs. Default "carbon.super". (optional)
+     * @param headers         Any additional headers to be embedded. (optional)
+     * @return Resend response.
+     * @throws ApiException If fails to make API call.
+     */
+    public ResendResponse resendPasswordNotification(ResendRequest resendRequest, String tenantDomain,
+                                                     Map<String, String> headers) throws ApiException {
+
+        final String localVarPath = "/password/resend";
+        return resend(resendRequest, tenantDomain, headers, localVarPath);
+    }
+
+    /**
+     * Confirm password recovery.
+     *
+     * @param confirmRequest   Password recovery confirm request. (required)
+     * @param tenantDomain     Tenant Domain which user belongs. Default "carbon.super". (optional)
+     * @param headers          Any additional headers to be embedded. (optional)
+     * @return Confirm response.
+     * @throws ApiException If fails to make API call.
+     */
+    public ConfirmResponse confirmPasswordRecovery(ConfirmRequest confirmRequest, String tenantDomain,
+                                                   Map<String, String> headers) throws ApiException {
+
+        final String localVarPath = "/password/confirm";
+        return confirm(confirmRequest, tenantDomain, headers, localVarPath);
+    }
+
+    /**
+     * Reset user password.
+     *
+     * @param resetRequest   Password reset request. (required)
+     * @param tenantDomain   Tenant Domain which user belongs. Default "carbon.super". (optional)
+     * @param headers        Any additional headers to be embedded. (optional)
+     * @return reset response.
+     * @throws ApiException If fails to make API call.
+     */
+    public ResetResponse resetUserPassword(ResetRequest resetRequest, String tenantDomain,
+                                           Map<String, String> headers) throws ApiException {
+
+        final String localVarPath = "/password/reset";
```

Comment:
Shall we take these static constants

Reply:
addressed with https://github.com/wso2/carbon-identity-framework/pull/5601/commits/c46dae55639b7b896f3389a9dd4b8980972dabb5